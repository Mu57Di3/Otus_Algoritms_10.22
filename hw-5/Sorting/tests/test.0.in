10
287,382,350,707,892,988,419,872,949,630
