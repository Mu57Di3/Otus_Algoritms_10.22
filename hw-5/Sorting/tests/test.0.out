287,350,382,419,630,707,872,892,949,988
